org.openrgb.OpenRGB.yaml

```diff
@@ -0,0 +1,40 @@
+app-id: org.openrgb.OpenRGB
+runtime:  org.kde.Platform
+runtime-version: '5.15'
+sdk: org.kde.Sdk
+command: openrgb
+finish-args:
+- --socket=x11
+- --device=all
+
+modules:
+- name: libusb
+  buildsystem: autotools
+  sources:
+  - type: git
+    url: 'https://github.com/libusb/libusb.git'
+    commit: '1a906274a66dd58bf81836db1306902d4a7dc185'
```

Comment:
Any reason to use unreleased git snapshots?

Reply:
we have a shared-module for this.